crates/bevy_reflect/bevy_reflect_derive/src/utility.rs

```diff
@@ -156,10 +156,10 @@ pub(crate) fn extend_where_clause(
         quote!()
     };
     generic_where_clause.extend(quote! {
```

Comment:
Nit: Could you leave a comment above this line letting future devs know that the parentheses are there to disambiguate certain types (specifically HRTBs)?

Reply:
Sure.

Reply:
Now that I look at it, I'm not sure where `parameter_types` is coming from. If it's coming from the user's own type parameters, I should remove the parentheses. But if it's generated as part of the derive, then I should leave it in.

Reply:
Okay no, they're coming from Bevy. The parentheses remain.

Reply:
Done and pushed.